Formula/speech-tools.rb

```diff
@@ -71,7 +71,7 @@ def install
 
     # extract one frame from the middle using ch_track, capturing stdout
     pitch = shell_output("#{bin}/ch_track #{ptcfile} -from #{frequency_hz * duration_secs / 2} " \
-                                                    "-to #{frequency_hz * duration_secs / 2}")
+                         "-to #{frequency_hz * duration_secs / 2}")
```

Comment:
We should probably just extract the command we want to run into a string and pass that to `shell_output`.